src/components/ui/check-chip-group.svelte

```diff
@@ -25,12 +25,13 @@
   }
 
   const dispatch = createEventDispatcher();
-  function checkLimit(evt) {
+  function checkLimit(evt, value) {
     if (maxChecked !== null && currentChecked + evt.target.checked > maxChecked) {
       evt.preventDefault();
-      return false;
+    } else {
+      value.checked = !value.checked;
```

Comment:
You'd think. But it didn't work without this line, so idfk

Reply:
Maybe it is about the order of calls. This is an event handler, so it makes sense it waits for it to return before changing the bound variable so that if the function calls `preventDefault` or returns false, it doesn't change the variable.

Reply:
Well, this works, I figured I wouldn't put too much thought to it)